tests/jenkins/TestOpenSearchDashboardsIntegTest.groovy

```diff
@@ -86,5 +93,45 @@ class TestOpenSearchDashboardsIntegTest extends BuildPipelineTest {
     void integTests_runs_consistently() {
```

Comment:
Rename to 
```suggestion
    void integTests_runs_for_all_components() {
```

Reply:
Per @Divyaasm this has been fixed in the PR.